bring-it-up/bring-it-up
# FE #32 & #34: Show live data on service card chips & Add home page banner

## Description
#### Summary:
Replace placeholder data on the service cards with live data for the fields:
- Support Type (ServiceType)
- Methods of delivery
- Specialties
- Target demographic (targetClients)

Also add a banner to the home page.

#### Changes:
- Replace placeholder data in `tags` array with live data from the service for `delivery`, `targetClients`, `serviceType`, and `specialty` 
- Change the styling of `tags`, `tag`, `cardContent`, and `serviceCard` to accommodate for more chips
- Add banner to home page 

#### How to test this PR:
- On the Home page, check that all cards have correct chips showing and consistent styling
- Check that the banner matches the designs: https://www.figma.com/file/EdBT6GXCgMweptmjlQ3egu/Bring-It-Up?type=design&node-id=411-73
  
## Checklist
